test/unit/TextPath-test.ts

```diff
@@ -277,7 +277,7 @@ describe('TextPath', function () {
     layer.add(textpath);
     stage.add(layer);
 
-    cloneAndCompareLayer(layer, 200);
+    cloneAndCompareLayer(layer, 200, 10);
```

Comment:
This one is interesting, here are screenshots from master:
![2023-03-22_20-10-22](https://user-images.githubusercontent.com/17948218/227041743-d7f66c02-3ecc-4dcf-9e0e-dff9792316d0.png)
![2023-03-22_20-10-26](https://user-images.githubusercontent.com/17948218/227041759-a4ed26cf-0d39-4fb4-ac69-176db0dfbe7e.png)
![2023-03-22_20-10-31](https://user-images.githubusercontent.com/17948218/227041775-ba63c169-6aa7-48bf-9e9f-8ba18aab33e5.png)

Here comes this PR:
![2023-03-22_20-10-48](https://user-images.githubusercontent.com/17948218/227041808-6c375210-084d-4a95-a94a-6556af288d2a.png)

![2023-03-22_20-10-51](https://user-images.githubusercontent.com/17948218/227041825-bbd96462-ec06-4783-95ae-fea1dc8e211e.png)

![2023-03-22_20-10-57](https://user-images.githubusercontent.com/17948218/227041846-2a92f560-0311-4e52-88a8-7904f8918762.png)

Adding a second tolerance value fixes it. I believe it is retry, right? It this acceptable?